src/panels/AssetPanel.php

```
@@ -217,6 +219,71 @@ protected function getToolbarItems(): array
+
+        return [
+            'baseUrl' => is_string($baseUrl) ? $baseUrl : '',
+            'devMode' => $devMode === true,
+            'devServerUrl' => is_string($devServerUrl) ? $devServerUrl : null,
+            'entries' => $entries,
+            'entrypoints' => is_array($entrypoints)
+                ? array_values(array_filter($entrypoints, is_string(...)))
+                : [],
+            'manifestPath' => $manifestPath,
+        ];
+    }
```

PHPMD (2.15.0)

[warning] The method captureVite() has a Cyclomatic Complexity of 15. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)